cmd/account/reset.go

```diff
@@ -114,25 +121,46 @@ func (o *resetOptions) run() error {
 			}
 		}
 	}
-
 	account, err := k8s.GetAWSAccount(ctx, o.kubeCli, o.accountNamespace, o.accountName)
 	if err != nil {
 		return err
 	}
-
+	//get accountID for reset
+	var accountId string
+	accountId = account.Spec.AwsAccountID
+	//	payerAccount = account.Spec.IAMUserSecret
 	// reset fields in spec
-	account.Spec.ClaimLink = ""
-	account.Spec.ClaimLinkNamespace = ""
-	account.Spec.IAMUserSecret = ""
+	//	account.Spec.ClaimLink = ""
+	//	account.Spec.ClaimLinkNamespace = ""
+	//	account.Spec.IAMUserSecret = ""
+	//awsClient, err := awsprovider.NewAwsClient("osd-staging-2", "us-east-1", "")
+	awsClient, err := o.getAwsClient("aws-account-operator-credentials", "aws-account-operator")
+	o.awsClient = awsClient
+	if err != nil {
+		return err
+	}
+	var rootId string
+	rootId, err = o.getRoot(o.awsClient)
```

Comment:
This function is both called as a method of resetOptions, and the member var awsClient is being passed. Given the request above, I think this can be reimplemented to be
`rootId, err :=  getRoot(&awsClient)`